e2e/rbd.go

```diff
@@ -57,6 +57,9 @@ var (
 	appBlockSmartClonePath = rbdExamplePath + "block-pod-clone.yaml"
 	snapshotPath           = rbdExamplePath + "snapshot.yaml"
 	defaultCloneCount      = 10
+
+	kernelRelease = ""
+	nbdMapOptions = "debug-rbd=20"
```

Comment:
would be good to define a constant here? no hard opinion though.

Reply:
Could you elaborate I doubt I got you?

Reply:
constant for debug level?

Reply:
If yes, Nah! this is good enough.